nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -56,6 +56,22 @@ in
         GZIP compression level of the resulting disk image (1-9).
       '';
     };
+
+    virtualisation.googleComputeImage.contents = mkOption {
+      type = with types; listOf attrs;
+      default = [ ];
+      description = ''
+        The files and directories to be placed in the target file system.
```

Comment:
nit: adding an example attribute would be nice, probably quicker to parse for at least some people.

(I also think "in the image" might be simpler and easier, as there's only one file system anyway.)